apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -588,46 +570,47 @@ private fun ChatList(chatModel: ChatModel, searchText: MutableState<TextFieldVal
   val searchChatFilteredBySimplexLink = remember { mutableStateOf<String?>(null) }
   val chats = filteredChats(showUnreadAndFavorites, searchShowingSimplexLink, searchChatFilteredBySimplexLink, searchText.value.text, allChats.value.toList())
   LazyColumnWithScrollBar(
-    Modifier.fillMaxWidth(),
-    listState
+    Modifier.fillMaxSize(),
+    listState,
+    reverseLayout = oneHandUI.state.value
   ) {
     stickyHeader {
       Column(
         Modifier
           .offset {
             val y = if (searchText.value.text.isEmpty()) {
+              val offsetMultiplier = if (oneHandUI.state.value) 1 else -1
               if (
                 (oneHandUI.state.value && scrollDirection == ScrollDirection.Up) ||
                 (appPlatform.isAndroid && keyboardState == KeyboardState.Opened)
               ) {
                 0
-              } else if (listState.firstVisibleItemIndex == 0) -listState.firstVisibleItemScrollOffset else -1000
+              } else if (listState.firstVisibleItemIndex == 0) offsetMultiplier*listState.firstVisibleItemScrollOffset else offsetMultiplier*1000
```

Comment:
Please, add space in this case like `offsetMultiplier * listState.....` to comform to our code style